samples/Catalog/package.json

```diff
@@ -1,13 +1,13 @@
 {
   "name": "Catalog",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
That would be ideally a good idea but highly impracticable. Consider the case when the Catalog needs to be upgraded and the package does not. 
I would bump the corresponding version only when a change takes places, but we can discuss.